packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -175,6 +176,7 @@ const ProofOfIdentityContainer = ({
             has_poa={has_poa}
             height={height ?? null}
             handleComplete={handleComplete}
+            is_message_enabled={is_message_enabled}
```

Comment:
You'll need to pass this from `proof-of-identity.jsx` as well. If not, this will always be `false`.

Reply:
my bad, i was twisted with enabled/disabled, have pushed the fix it, please help to rereview